src/esp/gfx/Viewer.cpp

```diff
@@ -26,7 +26,9 @@ namespace gfx {
 Viewer::Viewer(const Arguments& arguments)
     : Platform::Application{arguments,
                             Configuration{}.setTitle("Viewer").setWindowFlags(
-                                Configuration::WindowFlag::Resizable)},
+                                Configuration::WindowFlag::Resizable),
+                            GLConfiguration{}.setColorBufferSize(
+                                Vector4i(8, 8, 8, 8))},
```

Comment:
To explain further: for some reason, GLFW defaults to 8bit alpha while SDL2 defaults to 0 and even after trying multiple times to find the reason behind I still don't know why is it so. Since GLFW is desktop-only while SDL2 works also on phones and web, I'm afraid that having it 8bit everywhere might cause problems on phones and web. In particular, WebGL has quite some config options related to alpha and they have probably very good performance reasons why it isn't enabled by default. 

OTOH the `Invalid instruction 4` might mean that Metal no longer accepts 24bit framebuffer format in the latest macOS. Actually, checking their format tables, it seems it never did (only 8/16/32bit formats are listed) so it looks like it was only for compatibility with GL and that's another thing that silently broke in the new version. Or something like that.

Reply:
@mosra I built the Magnum tests and I can confirm that `PlatformSdl2ApplicationTest` works as expected (no crash due to `Invalid instruction 4`), whereas `PlatformGlfwApplicationTest` crashes with a backtrace as reported before (`Invalid instruction 4`).

Reply:
Oh well, that is exactly what I *did not want* to see :sweat_smile: Now I need to figure out why alpha=0 works on SDL but not on GLFW...